TestingTravis/TestClass.swift

```diff
@@ -17,6 +17,7 @@ class TestClass: NSObject {
         print(forceUnwrapMe!)
         print(forceUnwrap2!)
         //testing code owners
+        //test 2
         let unused = 0
     }
     
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Nakul97/TestingTravis/blob/e9e6ea06e61cf58b68cb2c3fd1e8ff5de51b4ab3/TestingTravis/TestClass.swift#L23"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TestClass.swift:23</code></td>
    </tr>
  </tbody>
</table>